### PR TITLE
Explore: support ANSI colors in live logs

### LINKED ISSUE #28893: Support ANSI colors in live logs
**What would you like to be added**:

Support for ANSI colors in the Logs Live view:

Selection_2020-11-06_399x21

There is currently support in the non-live view:

Selection_2020-11-06_392x25

**Why is this needed**:

For enhanced readability of live logs

### PR DESCRIPTION
**Pull request by avaly**
**What this PR does / why we need it**: Support for ANSI colors in the Logs Live view

**Which issue(s) this PR fixes**:

Fixes #28893

### CODE REVIEW

**Review by ivanahuckova — APPROVED** [review]
Thanks @avaly for this great contribution! LGTM! 🚀 Let's keep this open till tomorrow so other reviewers can have a look at it as well. If there won't be any issues, I'll merge tomorrow. :)

File: public/app/features/explore/LiveLogs.tsx [hunk 1]
```diff
@@ -3,6 +3,7 @@ import { css, cx } from 'emotion';
 import tinycolor from 'tinycolor2';
 
 import { Themeable, withTheme, getLogRowStyles, Icon } from '@grafana/ui';
+import { LogMessageAnsi } from '@grafana/ui/src/components/Logs/LogMessageAnsi';
```
Comment (zoltanbedi) [inline public/app/features/explore/LiveLogs.tsx:6]: Can you please export this component from grafan-ui and add it to the import above?

**Review by zoltanbedi — CHANGES_REQUESTED** [review]

File: public/app/features/explore/LiveLogs.tsx (hunk 1 above)
Comment (avaly) [reply public/app/features/explore/LiveLogs.tsx]: Done!

**Review by zoltanbedi — APPROVED** [review]
🎉

**Comment by ivanahuckova** [comment]
I tried to close-open this PR to try to make `continuous-integration/drone/pr` run, but it didn't work, so I merged the master and it seems that it worked. So as soon as that passes with ✅ we can merge. Again, thanks so much for this contribution. It is great!

**Pull request merged**